docs/src/datamanagment/metadata-search.md

```diff
@@ -58,24 +58,29 @@ Metadata Search works by enabling metadata indexing on selected repositories and
 where `repo` is the data repository id.  
 2. **Creates matching branches** in the metadata repository for each configured branch in the data repository. For example, a `dev`
 branch in data repository `my-repo`, will have a corresponding `dev` branch in `my-repo-metadata`.
-3. **Maintains a versioned Iceberg object metadata table** on each matching branch in the metadata repository at:
-`<repo>-metadata.<branch>.system.object_metadata`.
+3. **Maintains a versioned Iceberg object metadata table** on each matching branch in the metadata repository.
 4. **Continuously syncs metadata** via a background processing pipeline that keeps the object metadata tables
    [eventually consistent](#consistency) with changes in the corresponding data repository branches.
+5. **Wraps the lakeFS Iceberg REST catalog** to transparently redirect table queries to the correct physical location, so
```

Comment:
it translate the requests from the catalog on the lakefs repository the catalog on the metadata search repository.

Reply:
translates repository and reference (maybe `dereference` would be suitable here)